gtsam/linear/Sampler.h

```diff
@@ -63,15 +63,9 @@ class GTSAM_EXPORT Sampler {
   /// @name access functions
   /// @{
 
-  size_t dim() const {
-    assert(model_.get());
```

Comment:
Was there a reason this assert was added in the first place? Seems weird to have it from before. Sadly no documentation :(